### PR TITLE
minor modification

### PR DESCRIPTION
**Pull request by joinimran**
Made a correction in the statement. 

Problem: https://github.com/MicrosoftDocs/windows-itpro-docs/issues/8568

### CODE REVIEW

**Comment by joinimran** [comment]
@JohanFreelancer9 - kindly do the copy editor review for this PR. Thanks.

File: windows/security/threat-protection/microsoft-defender-atp/apis-intro.md [hunk 1]
```diff
@@ -57,7 +57,7 @@ You can access Microsoft Defender ATP API with **Application Context** or **User
 - **User Context:** <br>
     Used to perform actions in the API on behalf of a user.
 
-	Steps that needs to be taken to access Microsoft Defender ATP API with application context:
+	Steps that needs to be taken to access Microsoft Defender ATP API with user context:
```
Comment (JohanFreelancer9) [inline windows/security/threat-protection/microsoft-defender-atp/apis-intro.md:60]:
```suggestion
	Steps that need to be taken to access Microsoft Defender ATP API with user context:
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Comment by joinimran** [comment]
Suggestions applied.

**Review by denisebmsft — APPROVED** [review]
Thank you @joinimran

**Pull request merged**